xml/System.Security.Cryptography.X509Certificates/X500DistinguishedNameBuilder.xml

```diff
@@ -14,8 +14,26 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>This class facilitates building a distinguished name for an X.509 certificate.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When constructing the SEQUENCE OF Relative Distinguished Names, this builder constructs the SEQUENCE OF in the opposite order which they were added to the builder. For example:
+
+```cs
+builder.AddCommonName("Contoso");
+builder.AddCountryOrRegion("US");
+```
+
+Will produce a SEQUENCE OF with the country or region first, and the common name second. Because <xref:System.Security.Cryptography.X509Certificates.X500DistinguishedName> follows RFC 4514
+when converting a Distinguished Name to a string, it starts with the last Relative Distinuished Name component, moving backward toward the first.
+
+Because the builder creates the SEQUENCE OF in reverse, and <xref:System.Security.Cryptography.X509Certificates.X500DistinguishedName> is also in reverse, it gives the appearence of the added attributes and the string representation of the Distinguished Name in the same order.
```

Comment:
```suggestion
Will produce a `SEQUENCE OF` with the country or region first, and the common name second. Because <xref:System.Security.Cryptography.X509Certificates.X500DistinguishedName> follows RFC 4514
when converting a `Distinguished Name` to a string, it starts with the last `Relative Distinguished Name` component, moving backward toward the first.

Because the builder creates the `SEQUENCE OF` in reverse, and <xref:System.Security.Cryptography.X509Certificates.X500DistinguishedName> is also in reverse, it gives the appearance of the added attributes and the string representation of the `Distinguished Name` in the same order.
```